documentation/tutorial.md

```diff
@@ -332,7 +361,24 @@ end
 
 ```
 
-If you run this spider `Crawly.Engine.start_spider(Homebase)`, it will output the extracted data with the log:
+You will also need to tell Crawly where to store the scraped data. Create `config/config.exs` file with the following
+contents:
+```elixir
+use Mix.Config
+
+config :crawly,
+  pipelines: [
+    Crawly.Pipelines.JSONEncoder,
+    {Crawly.Pipelines.WriteToFile, folder: "/tmp", extension: "csv"}
+  ]
+```
+The scraped data will now be stored in a CSV file under `/tmp` directory on your filesystem. The name of the file
+will be the same as our spider name - in our case `Homebase.csv`.
+
+If you restart iex and run this spider `Crawly.Engine.start_spider(Homebase)` again, it will output the extracted data 
+with the log:
+
+TODO: I can't see any "Scraped" messages like these below. Has that been removed on purpose or is a bug?
```

Comment:
I think you're right, please remove these messages, as they do not show up anymore

Reply:
👍 will do